documentation/modules/tracing/proc-configuring-jaeger-tracer-kafka-clients.adoc

```diff
@@ -21,23 +25,50 @@ In each client application:
     <version>{JaegerClientVersion}</version>
 </dependency>
 ----
++
+.Dependencies for Jaeger tracing with OpenTelemetry
+[source,xml,subs="attributes+"]
+----
+<dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-sdk-extension-autoconfigure</artifactId>
+    <version>{OpenTelemetryVersion}</version>
+</dependency>
+<dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporter-jaeger</artifactId>
+    <version>{OpenTelemetryVersionStable}</version>
+</dependency>
+<dependency>
+    <groupId>io.grpc</groupId>
+    <artifactId>grpc-netty-shaded</artifactId>
+    <version>{OpenTelemetryGrpcVersion}</version>
+</dependency>
+----
 
-. Define the configuration of the Jaeger tracer using the xref:ref-tracing-environment-variables-{context}[tracing environment variables].
+. Define the configuration of the tracer using the xref:ref-tracing-environment-variables-{context}[tracing environment variables].
 
-. Create the Jaeger tracer from the environment variables that you defined in step two:
+. Create the tracer from the environment variables that you defined in step two.
++
+.Creating a tracer for OpenTelemetry
+[source,java,subs=attributes+]
+----
+OpenTelemetry ot = GlobalOpenTelemetry.get()
```

Comment:
What are we trying to achieve here? Tracing Kafka messages or adding some tracing to the application which is using the Kafka client? In the first case we don't need this code. In the second case we would then need to show how to use it?